packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -298,17 +302,15 @@ class Auth {
             this.set('customer_id', customerId)
             this.set('usid', usid)
             this.set('customer_type', isGuest ? 'guest' : 'registered')
-            this.pendingToken = Promise.resolve(this.data)
-            return this.pendingToken
+            return Promise.resolve(this.data)
```

Comment:
```suggestion
            return this.data
```
The return value from an async function is automatically wrapped in `Promise.resolve`, so you don't have to do it yourself.